packages/vhd-lib/src/Vhd/VhdAbstract.js

```diff
@@ -164,4 +167,40 @@ export class VhdAbstract {
       }
     }
   }
+
+  static async rename(handler, sourcePath, targetPath) {
+    await handler.rename(sourcePath, targetPath)
+  }
+
+  static async unlink(handler, path) {
+    const resolved = await resolveAlias(handler, path)
+    try {
+      await handler.unlink(resolved)
+    } catch (err) {
+      if (err.code === 'EISDIR') {
+        await handler.rmtree(resolved)
+      } else {
+        throw err
+      }
+    }
+
+    // also delete the alias file
+    if (path !== resolved) {
+      await handler.unlink(path)
+    }
+  }
+
+  static async createAlias(handler, aliasPath, targetPath) {
+    if (!isVhdAlias(aliasPath)) {
+      throw new Error(`Alias must be named *.alias.vhd,  ${aliasPath} given`)
+    }
+    if (isVhdAlias(targetPath)) {
+      throw new Error(`Chaining alias is forbidden ${aliasPath} to ${targetPath}`)
+    }
+    const aliasDir = path.resolve('/', path.dirname(aliasPath))
+    // only store the relative path from alias to target
+    const relativePathToTarget = path.relative(aliasDir, targetPath)
```

Comment:
Does not properly work ig `targetPath` does not start with `/`.

All our paths are virtual paths, separate from the local filesystem, and unless explicitly stated, are always absolute (because there is no notion of current working dir, relative paths are relative to the root).

Which is not the case for `path.resolve()` (and therefore `path.relative()` which uses it).